src/doc/rustc/src/profile-guided-optimization.md

```diff
@@ -145,3 +145,25 @@ in Clang's documentation is therefore an interesting read for anyone who wants
 to use PGO with Rust.
 
 [clang-pgo]: https://clang.llvm.org/docs/UsersManual.html#profile-guided-optimization
+
+## Community Maintained Tools
+
+As an alternative to directly using the compiler for Profile-Guided Optimization,
+you may choose to go with `cargo-get`, which has an intuitive API and saves you
```

Comment:
`cargo-pgo`, not `cargo-get` :)
Also I would say it's a CLI (command-line interface, not an API).

Reply:
API, as in the interface the user is talking to (the CLI options and flags), not necessarily HTTP API.